### ISSUE TITLE
Make a bootstrap template (no frills) 

### ISSUE THREAD

**Issue by rasmuse** [demokratikollen/demokratikollen#56]
Based on the suggestions made in issue #53.

**Comment by joelgoop** [comment]
I'd like to contribute to this (at least the technical bits) as well. Do you want to meet and discuss a little?

**Comment by rasmuse** [comment]
Yes, I would love to. I'll call you.

**Comment by joharohl** [comment]
Just out of curiosity, is the compass thing ment to be used for deving out the compiled css that then go into the Flask app? Or is it compiled when the app is running? 

Just checking if I have to think of it while doing the deploy. 

PS. ( There are ways to compile sass and coffee script using the Flask-Asset extension but I haven't looked much at it )

**Comment by rasmuse** [comment]
No, at least I was thinking that it's up to the frontend developer to put
templates and css into the flask app folder.

This means one thing less to worry about in deploy, and most likely a
faster testing cycle for frontend developers. After making changes to
template files and sass files we just need to do `compass compile [...]`
and the site will be updated on the development machine. (If/when we have
some more steps, we might create a Gruntfile for it).

**Comment by joelgoop** [comment]
Played around a little with the template, so if you're going to work on it (@rasmuse), pull first!

**Comment by joelgoop** [comment]
Regarding content markup for search engines, Facebook, etc., a quick search gave a few sites that might be good to read and have in mind when designing object pages.

For schema.org markup used by Google and others to present "rich" search results:
http://schema.org/docs/gs.html

To prepare sites for Facebook sharing:
https://developers.facebook.com/docs/web/webmasters/

**Comment by rasmuse** [comment]
Thanks @joelgoop. Content markup for search engines is a high priority for 1.0 in my eyes. Should we make an issue for it?

A little bit of status update: I fiddled around in the `bootstrap-template` branch trying to improve the application structure while I was also creating some basic setup of tests and pages. More info on the overall structure in #67. Further to that, you'll also see I have started breaking the `layout.html` apart into reusable parts. Each folder under `templates` has a template called `base.html`, and my idea is that all templates in a folder should inherit the local `base.html` template. So if you want a simple page with the menu item "Riksdagen" highlighted, then make your new template at `templates/parliament/new_template.html`, and start with `{% extends "parliament/base.html" %}`.

I've said this a few times now, but now I _really_ think I am ready to design a mockup of the "Member of parliament" page. I'll try to get something done tomorrow evening.
